src/main/java/duke/iomethods/IOMethods.java

```diff
@@ -0,0 +1,97 @@
+package duke.iomethods;
+
+import duke.exception.DukeException;
+
+import java.util.ArrayList;
+
+public class IOMethods {
+    public static ArrayList<String> splitToTwo(String line, String delimiter) {
+        ArrayList<String> words = new ArrayList<>();
+        int divider = line.indexOf(delimiter);
+
+        words.add(line);
+
+        if (divider != -1) {
+            words.set(0, line.substring(0, divider));
+            words.add(line.substring(divider+1));
+        }
+        return words;
+    }
+
+    public static void printWithDivider(String stringWithinDivider) {
+        String breakLine = "\t____________________________________________________________";
+        System.out.println(breakLine);
+        stringWithinDivider = stringWithinDivider.replace("\n", "\n\t");
+        System.out.println("\t" + stringWithinDivider);
+        System.out.println(breakLine);
+    }
+
+    public static void errorHandler(String input) throws DukeException {
+        String[] words = input.split(" ");
+        String command = words[0];
+
+        switch (command) {
+        case "todo":
+        case "event":
+        case "deadline":
```

Comment:
You can refer to [switch case statement layout](https://se-education.org/guides/conventions/java/basic.html#layout) in the Java coding standards and add fallthrough comments here.